### PR TITLE
Style School Info Container

### PR DESCRIPTION
**Pull request by wwohlers**
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/XXXXXXXX/pulses/XXXXXXX)

Styles/fixes school info container.

## Screenshots

school-info-edit
school-info-no-edit

### CODE REVIEW

**Review by justinkonecny — APPROVED** [review]

File: src/containers/schoolInfo/SchoolInformationForm.tsx [hunk 1]
```diff
@@ -43,36 +38,40 @@ const SchoolInformationForm: React.FC<SchoolInformationFormProps> = ({
...
+              <Form.Item name="phone">
+                <Input disabled={!editMode} placeholder="Phone Number" />
+              </Form.Item>
               <Form.Item name="libraryStatus">
                 <Select disabled={!editMode} placeholder="Library Status">
-                  {Object.keys(LibraryStatus).map((key: string) => (
+                  {Object.entries(LibraryStatus).map(([key, value]) => (
                     <Option key={key} value={key}>
-                      {key}
+                      {value}
```
Comment (maxsebso) [inline src/containers/schoolInfo/SchoolInformationForm.tsx:74]: Yeah this is prob better

File: src/utils/libraryStatus.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 export const LibraryStatus = {
-  UNKNOWN: 'UNKNOWN',
-  EXISTS: 'EXISTS',
-  DOES_NOT_EXIST: 'DOES_NOT_EXIST',
+  UNKNOWN: 'Unknown',
+  EXISTS: 'Exists',
+  DOES_NOT_EXIST: 'Does Not Exist',
```
Comment (maxsebso) [inline src/utils/libraryStatus.tsx:4]: This enum was meant to match backend response. Are we sure it still works?

**Review by maxsebso — COMMENTED** [review]
As long as you confirm that changing the enum does not mess up the connection to the backend, it looks good to me.

**Review by maxsebso — APPROVED** [review]
LGTM

**Pull request merged**